vm_insnhelper.c

```diff
@@ -4379,6 +4379,12 @@ vm_concat_array(VALUE ary1, VALUE ary2st)
     return rb_ary_concat(tmp1, tmp2);
 }
 
+VALUE
+rb_vm_concat_array(VALUE ary1, VALUE ary2st)
+{
+    return vm_concat_array(ary1, ary2st);
+}
+
```

Comment:
I think you can save a C function call here by directly binding vm_concat_array rather than wrapping it in an extra layer.

Reply:
The wrapping is necessary because `vm_concat_array` is static. Might want to comment that it's for YJIT, though.

Reply:
Yep, we had to wrap it cause the original function was static - following `splatarray` convention from [this change](https://github.com/Shopify/ruby/commit/be648e0940054a693105b1a3da6c2675ee724499#diff-f8c174347e6ea8889b5036064a1ff4fe5e7c53a821befa9bdc5ccbf17800a649R4261). I will add a comment!

Reply:
Ah, I'd forgotten that we included vm*.c into yjit.c.